browser/css/jsdialogs.css

```diff
@@ -568,6 +573,25 @@ td.jsdialog > [id^='table-box']:not(.sidebar) {
 	/* grid-column: '1 / ' + (this._columns + 1) - set inside JS */
 }
 
+#ServerAuditDialog .ui-treeview-headers {
+	display: grid;
+	grid-template-columns: auto 1fr 170px;
+}
```

Comment:
I see 2 problems here:

* I would prefer to avoid hard coded pixels and instead use ratios
* Where I'm missing a pixel value is on the initial column (the icon)

```css
grid-template-columns: var(--btn-img-size-s) 6fr 3fr;
```

Reply:
Used `0fr` to remove the unused icon